### DISCUSSION TITLE
CMD/alist server后有警告提示，怎么解决

### DISCUSSION THREAD

**Discussion by jitu1973** [AlistGo/alist#2945] [Invalid question]
第一次按照网上的教程搭建
在CMD里alist server后有警告提示，如图
![Snipaste_2023-01-07_23-35-22](https://user-images.githubusercontent.com/122166792/211158877-621279a5-c553-4b80-9746-5e3b6d51c839.png)

**Comment by NovaShen0724** [comment]
1.有问题请Q&A
2.你没有建立索引当然不能搜索啦，你去后台新建索引再重启看看是不是没有警告

**Comment by 12189108** [comment]
能正常用就无视，其他上面的仁兄已经讲了

**Comment by Nu-Exception** [comment]
提示：load config error: readObjectStart: expect { or n, but found , error found in #1 byte of ...||..., bigger context ...||... 呢，怎么解决？

**Reply by fm365** [reply]
我也碰到这个问题， 没解决
